src/pages/Home/container.js

```diff
@@ -15,7 +15,7 @@ Home.propTypes = {
 };
 
 const mapStateToProps = (state) => ({
-  userLoggedIn: isUserLoggedIn(state)
+  userLoggedIn: isUserLoggedIn(state) || ''
```

Comment:
Make this condition inside `isUserLoggedIn` selector, this will work for all places where you use this function and you don't need to add this condition to each mapStateToProps

Reply:
Fixed